style.css

```diff
@@ -1,6 +1,8 @@
 @import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
 
 * {
+  padding: 0;
+  margin: 0;
   box-sizing: border-box;
 }
```

Comment:
- [ ] For a great user experience, kindly make sure you stick to the design as much as possible. (typos, spacing, color)

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201910509-1f241b2d-376d-4cd0-8f13-b737504e60f3.png)
Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201910888-b8e4dba1-ec6e-423e-ba5b-98a8511b54cd.png)